src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/DuplicateFinder.java

```diff
@@ -29,7 +29,12 @@ public DuplicateFinder(
             .listBlobs()
             .stream()
             .filter(b -> isDuplicate(b.getName(), containerName))
-            .map(b -> new Duplicate(b.getName(), containerName, b.getProperties().getLastModified().toInstant()))
+            .map(b -> new Duplicate(
+                    b.getName(),
+                    containerName,
+                    b.getProperties().getLastModified().toInstant(),
+                    b.getProperties().getContentLength()
```

Comment:
which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()`